app/jobs/push_handover_dates_to_delius_job.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class PushHandoverDatesToDeliusJob < ApplicationJob
+  queue_as :default
+
+  def perform(nomis_offender_id)
+    offender = OffenderService.get_offender(nomis_offender_id)
```

Comment:
It looks like this job does too much. Actually we had the offender object in our hands in the ProcessDeliusDataJob#import_data - seems a shame to go and get it again? Maybe this job doesn't need to exist (but I guess it does because of the rake task) - now I'm not sure...